docs/sources/tutorials/api_org_token_howto.md

```diff
@@ -71,4 +72,4 @@ Using the Token that was created in the previous step, you can create a dashboar
   }' http://localhost:3000/api/dashboards/db
   ```
 
-  This import will not work if you exported the dashboard via the Share -> Export menu in the Grafana UI (it strips out data source names etc.). View the JSON and save it to a file instead or fetch the dashboard JSON via the API.
+  This import will not work if you export the dashboard using the Share -> Export menu in the Grafana UI (it strips out data source names etc.). View the JSON and save it to a file instead or fetch the dashboard JSON through the API.
```

Comment:
This is an improvement, but I'm thinking we can do even more with this paragraph. 

Maybe something like this?

_Note that it is not possible to import a dashboard that was exported using the Share -> Export menu in the Grafana UI. Instead, either select View JSON, and save it to a file, or fetch the JSON through the API._

Reply:
Actually, since an update while ago in Share -> Export you have to actively select that you want to export a templated dashboard, otherwise it will behave the same as "View JSON", for reference:
![image](https://user-images.githubusercontent.com/1668778/63943911-daf2bd80-ca70-11e9-8dc8-c98a9a8613e5.png)

So maybe we could improve this some more by referring to " import a dashboard that was exported to be shared externally" or something similar?

Reply:
Updated the note as suggested by reviewers.